homeassistant/components/services.yaml

```diff
@@ -607,4 +607,14 @@ homeassistant:
     fields:
       entity_id:
         description: The entity_id of the device to turn off
-        example: light.living_room
\ No newline at end of file
+        example: light.living_room
+
+
+remember_the_milk:
```

Comment:
Please move this to `remember_the_mil/services.yaml`.

Reply:
OK